src/qmapshack/gis/rte/router/CRouterRoutino.h

```diff
@@ -1,5 +1,5 @@
 /**********************************************************************************************
-    Copyright (C) 2014 Oliver Eichler <oliver.eichler@gmx.de>
+    Copyright (C) 2014 Oliver Eichler oliver.eichler@gmx.de
```

Comment:
same as in the cpp file.

Reply:
OK do I do another fix and then pull request ...